api/task_processor/management/commands/checktaskprocessorthreadhealth.py

```diff
@@ -0,0 +1,17 @@
+import logging
```

Comment:
Because, by default, django management commands use the name of the module as the argument to pass when calling `python manage.py <command>` and my understanding (based on e.g. `python manage.py makemigrations|runserver`) is that these commands should be formatted like this.

<img width="318" alt="image" src="https://user-images.githubusercontent.com/14089968/185196451-c30da6d4-091e-4d61-9b2e-cbe490db194d.png">